docs/StardustDocs/topics/gettingStarted.md

```diff
@@ -1,22 +1,170 @@
+## Install Kotlin
 
-## Feedback and contributing
-If you found a bug, or have an idea for a new feature, please [file an issue](https://github.com/Kotlin/dataframe/issues/new) in DataFrame GitHub repository.
+Kotlin is included in each IntelliJ IDEA release.
+Download and install [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) to start using Kotlin.
 
-If you wish to contribute, you're welcome! Choose an issue you like, let us know that you're working on it, and prepare a [pull request](https://github.com/Kotlin/dataframe/pulls). We'll review and merge it if everything goes well.
+## Create Kotlin project
 
-You can also give us feedback or ask a question in [#datascience](https://kotlinlang.slack.com/archives/C4W52CFEZ) channel of a Kotlin Slack.
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location, if necessary.
 
-Good luck!
+   > Select the **Create Git repository** checkbox to place the new project under version control. You can enable this
+   > later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+5. Select the **Gradle** build system.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+7. From the **Gradle DSL** list, select **Kotlin** or **Groovy**.
+8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
+9. Click **Create**.
+
+You have successfully created a project with Gradle.
+
+### Update Gradle dependencies
+
+In your Gradle build file (`build.gradle.kts`), add the DataFrame library as a dependency:
+
+<tabs>
+<tab title="Kotlin DSL">
+
+```kotlin
+dependencies {
+    implementation("org.jetbrains.kotlinx:dataframe:%dataFrameVersion%")
+}
+```
+
+</tab>
+
+<tab title="Groovy DSL">
+
+```kotlin
+dependencies {
+    implementation 'org.jetbrains.kotlinx:dataframe:%dataFrameVersion%'
+}
+```
+
+</tab>
+
+</tabs>
+
+### Add imports
+
+In `src/main/kotlin/Main.kt`, add the following imports at the top of the file:
+
+```kotlin
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.io.*
+import org.jetbrains.kotlinx.dataframe.api.*
+```
+
+## Import data
+
+Download and save our <res resource-id="movie-sample-data"> file to the root directory of your project.
+
+Delete the `println()` functions and comments from your main function in `Main.kt`.
```

Comment:
Probably better to avoid create empty file with println() function and directly suggest the addition of the file, say, myDataFrameHelloWorld.kt with the template piece of code